courses/free-courses-en.md

```diff
@@ -618,11 +619,11 @@
 * [Web Development By Doing: HTML / CSS From Scratch](https://www.udemy.com/course/web-development-learn-by-doing-html5-css3-from-scratch-introductory/) - Udemy
 * [What the Flexbox](https://flexbox.io) - Wesbos
 
-
```

Comment:
Don't delete this line, there should be a blank lines between last section and next heading.